File: ParteJava/ProyectoPro3/ProyectoPro3DBManager/src/main/java/pe/edu/pucp/proyectopro3/db/MySQLDBManagerFactory.java
package pe.edu.pucp.proyectopro3.db;

public class MySQLDBManagerFactory extends DBManagerFactory {
    @Override
    public DBManager crearDBManager(String host, int puerto, String esquema,
                                    String usuario, String password) {
        return MySQLDBManager.getInstance(host, puerto, esquema, usuario,
                password);
    }
}

File: ParteC#/ProyectoProgramacion3/ProyectoProgramacion3DBManager/DB/MySqlDbManagerFactory.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ProyectoProgramacion3DBManager.DB;
public sealed class MySqlDbManagerFactory : DbManagerFactory
{
    public override DbManager CrearDbManager(string connectionStringBase)
    {
        return MySqlDbManager.GetInstance(connectionStringBase);
    }
}
